DigitalLearningSolutions.Web/Helpers/CustomClaimHelper.cs

```diff
@@ -25,15 +25,21 @@ public static int GetCentreId(this ClaimsPrincipal user)
             return user.GetCustomClaimAsRequiredInt(CustomClaimTypes.UserCentreId);
         }
 
+        /// <summary>
+        ///     Returns the Admin Category ID or null if the ID is non-existent
+        ///     Also returns null if ID is zero to match the data service convention of not filtering on NULL category filter
+        /// </summary>
         public static int? GetAdminCategoryId(this ClaimsPrincipal user)

```

Comment:
Can we rename the method as well? To something like GetAdminCourseCategoryFilter?
My problem with the current name is that you're giving the concept of "AdminCategoryId" two definitions - one when used in a databasey context and another when used in a business-logic-y context.